<launch>
    <!-- Arguments -->
    <arg name="centroid_topic" default="/centroid_markers" />
    <arg name="intent_prediction_topic" default="/object_id_to_grasp" />
    
    <!-- Frame IDs -->
    <arg name="target_frame" default="cam1_depth_optical_frame" />
    <arg name="source_frame" default="cam2_depth_optical_frame" />

    <!-- Node for Intent Prediction -->
    <node pkg="intent_pred_pkg" type="intent_prediction_node" name="intent_prediction_node" output="screen">
        <!-- Topics -->
    </node>
</launch>